docs/api.rst

```diff
@@ -12258,3 +12258,85 @@ Refresh general cache
   :statuscode 200: Caches were correctly updated
   :statuscode 409: An issue during refreshing caches occured
   :statuscode 500: Internal rotki error
+
+Getting Metadata For Airdrops
+========================
+
+.. http:get:: /api/(version)/blockchains/eth/airdrops/metadata
```

Comment:
we have aidrops in other chains for example the cow airdrop in gnosis. I would use

```suggestion
.. http:get:: /api/(version)/airdrops/metadata
```

since the airdrop page is chain agnostic